src/api/endpoints/index.js

```diff
@@ -0,0 +1,12 @@
+import { BASE_URL } from '../../config';
```

Comment:
Convention:
We usually don't have files containing a function only to return the url string, but a function that returns the axios (XHR generally) Promise.
E.g.
```js
export const getUserEngagements = (id) => axios.get(`${COMPANY_DATA(id)}/user-engagements`);
```
Separation and abstraction of functionality is good to some degree, but there can always be too much of a good thing.